config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml

```diff
@@ -3,6 +3,10 @@ periodics:
 - name: periodic-api-review-help
   interval: 1h
   decorate: true
+  annotations:
+    testgrid-dashboards: sig-testing-fejtabot
+    description: Adds API review process description to kind/api-change PRs
+    testgrid-tab-name: api-review-help
```

Comment:
sort these keys?

Reply:
They're "sorted": mandatory to optional, and then alphabetically by key. What order would you prefer them in?

Also please see #13413 if there's a good reason that these should be in a particular order; this order's common among all the other annotations I wrote